src/plugins/intel_cpu/src/emitters/tpp/x64/jit_eltwise_emitters.cpp

```diff
@@ -0,0 +1,153 @@
+// Copyright (C) 2020-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "jit_eltwise_emitters.hpp"
+
+#include <cpu/x64/jit_generator.hpp>
+
+#include "snippets/lowered/port_connector.hpp"
+#include "snippets/lowered/port_descriptor.hpp"
+#include "transformations/snippets/tpp/op/eltwise.hpp"
+#include "snippets/lowered/port_descriptor.hpp"
+
+using namespace Xbyak;
+using namespace dnnl::impl;
+using namespace dnnl::impl::cpu::x64;
```

Comment:
I think it's useless as well :)

Reply:
They're not anymore 🙂 
Removed redundant namespaces in the constructors